src/Calendar.js

```diff
@@ -679,6 +685,19 @@ class Calendar extends React.Component {
     );
   }
 
+  handleRangeChange = (date, view) => {
+    let { onRangeChange } = this.props
+    if(onRangeChange) {
+      let range = VIEWS[view].range(date, {})
+      if(Array.isArray(range)) {
+        onRangeChange({ start: range[0], end: range[range.length -1]})
+      }
+      else{
+        onRangeChange(range)
```

Comment:
hmm I think if it's an array we want to pass all the values through, in the case of TimeGrig it maybe that there are "holes" in the range which is important to know. I'd just pass whatever the value is to `onRangeChange` and note in the doc comment that it's either an array or object for BULTIN views and YOLO for custom ones